docker/Dockerfile.contrib

```diff
@@ -24,21 +24,21 @@ USER root
 RUN apt-get update && apt-get install -y jq
 USER node
 WORKDIR /home/node/node-zwave-js
-RUN rm -f package-lock.json
+RUN rm -f yarn.lock
```

Comment:
Do you really need to remove the lockfile? This will cause pain when someone on the contrib image wants to update configs.

Reply:
They was removing package-lock.json, dunno the reason. I just did that to keep it unchanged